kazel/generator.go

```diff
@@ -17,19 +17,23 @@ limitations under the License.
 package main
 
 import (
-	"bytes"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"regexp"
 	"sort"
 	"strings"
 )
 
 const (
-	openAPIGenTag = "// +k8s:openapi-gen"
+	openAPITagName = "openapi-gen"
+	staging        = "staging/src/"
+)
 
-	staging = "staging/src/"
+var (
+	// Generator tags are specified using the format "// +k8s:name=value"
+	genTagRe = regexp.MustCompile(`//\s*\+k8s:([^\s=]+)?(?:=(\S+))?`)
```

Comment:
does this have to be on an empty line alone? maybe prepend `^\s*`?

Reply:
and append `$`?

Reply:
using the multiline flag (so `^` matches at the beginning of a line) slows this down to ~5-6 seconds to run.

adding `\n\s*` as a prefix makes this take about 2.3s, so a bit more than half a second slower.

adding `\s*\n` as a suffix doesn't seem to slow things down, though.

optimizing regular expressions is great.